app/models/category.rb

```diff
@@ -1,3 +1,5 @@
 class Category < ApplicationRecord
   has_many :tasks
+
+  validates :name, presence: true
 end
```

Comment:
Any chance we'd want a task to have multiple categories? If so, we could create a has_many through.